### ISSUE TITLE
Warning message "remote counter is unavailable".

### ISSUE THREAD

**Issue by philcollins012345** [custom-components/remote_homeassistant#279]
I'm using a shared counter between two instances using "Remote Assistant". All this works fine, but I get constantly (several times per minute) the following kind of messages in the log : "2024-04-03 10:50:14.992 WARNING (MainThread) [homeassistant.helpers.service] Referenced entities counter.h_mirror_l_power are missing or not currently available". Obviously the conter is available on both side and is updated almost in real time, so it can only be unavailable.  
Why is HA checking this conter all the time, while it is used only in one template running more or less every minute ?
It's only a warning message, but it fills the log quite fast.
The timestamp of this warning message is not directly correlated with the moment the counter is updated.

**Comment by Audiothor** [comment]
I've exactly the same issue. I didn't find any solution :(

**Comment by philcollins012345** [comment]
Did solve this issue on my side.
Initially I wanted to update a counter on both sites (the remote and the local one) but I discovered that HA doesn't like one does this.
I solved the issue by calling a remote service (in my case counter.increment) which updates the counter on the remote installation and I then get the feedback on the local instance of HA via the remote replication.
Remote service can be declared in the remote integration in order that one can use them from the local instance.
Hoping this can help.
